ospf6d/ospf6_neighbor.h

```diff
@@ -104,6 +104,10 @@ struct ospf6_neighbor {
 
 	/* BFD information */
 	struct bfd_session_params *bfd_session;
+
+	uint32_t seqnum_h;
+	uint32_t seqnum_l;
```

Comment:
How do those relate to the same name fields in `ospf6_auth_cfg`?  (Please add a comment)

Reply:
Its as per RFC7166. 32bit higher sequence number and 32 bit lower sequence number.

Reply:
That's not what I asked.  I asked what the difference is between `seqnum_h/l` in `ospf6_neighbor` and `seqnum_h/l` in `ospf6_auth_cfg`?

Reply:
seqnum_h/l in auth_cfg is used to send auth trailer header to neighbor.

seqnum_h/l in ospf6_neighbor is used to compare sequence number in received packet header.

Reply:
Please put this in a comment in the code, it's necessary to understand.